layers/meta-resin-raspberrypi/recipes-bsp/rpi-u-boot-scr/rpi-u-boot-scr/boot.cmd.in

```diff
@@ -1,8 +1,14 @@
+env delete resin_kernel_loader
 setenv resin_kernel_load_addr ${kernel_addr_r}
 run resin_set_kernel_root
 fdt addr ${fdt_addr}
 fdt get value bootargs /chosen bootargs
 env set bootargs "${bootargs} ${resin_kernel_root} rootwait"
 fdt rm /chosen bootargs
-fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${resin_kernel_load_addr} @@KERNEL_IMAGETYPE@@
+if test -n ${resin_kernel_loader}; then
+  env set resin_dev_bootcmd ${resin_kernel_loader} ${resin_dev_type} ${resin_dev_index}:${resin_root_part} ${resin_kernel_load_addr} /boot/@@KERNEL_IMAGETYPE@@ ;
+else;
+  env set resin_dev_bootcmd fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${resin_kernel_load_addr} @@KERNEL_IMAGETYPE@@;
+fi;
+run resin_dev_bootcmd
```

Comment:
Similar to https://github.com/resin-os/meta-resin/pull/1153#discussion_r214987891
I'm thinking it might be simpler to just move the kernel instead of having this check/attempt at backwards compatibility..

Reply:
I tend to agree as well. I don't see the benefit of having this backwards compatible as long as we don't have rollbacks yet.